### DISCUSSION TITLE
info oksigen semarang #2

### DISCUSSION THREAD

**Discussion by marthatyas** [kawalcovid19/wargabantuwarga.com#177] [General]
Terkait info oksigen di semarang. Agen oksigen atas nama AWC oksigen, no telpon yg berhasil saya hubungi adalah no 081325465932, no ini lebih fast respon (by phone) daripada no yang tertera. 
Selain itu alamat yang benar adalah Jalan Seruni VII No 9, Tlogosari Kulon, Kec Pedurungan, Semarang. 
Pekan lalu masih ada bisa refill hari senin dan rabu.

**Comment by zainfathoni** [comment]
Terima kasih atas laporannya Mbak @marthatyas, laporan Anda telah kami teruskan kepada tim terkait.

Untuk ke depannya, apabila Anda hendak mengoreksi informasi yang tertera di website kami, silakan klik tombol `Koreksi` di [halaman kontak yang ingin Anda koreksi](https://www.wargabantuwarga.com/provinces/jawa-tengah-14/awcoksigen-54).
Terima kasih. 🙏 

<img width="601" alt="Screenshot 2021-07-20 at 00 21 30" src="https://user-images.githubusercontent.com/6315466/126193491-b5938a27-a8dd-45ce-9a4a-3e48aa40ca71.png">
